httpcore/_backends/sync.py

```diff
@@ -137,10 +137,9 @@ def open_tcp_stream(
         exc_map = {socket.timeout: ConnectTimeout, socket.error: ConnectError}
 
         with map_exceptions(exc_map):
-            local_addrport = None
```

Comment:
`local_addrport` was unused, leftover?

Reply:
Yes, leftover from moving this code to computing `source_address` above. :+1: